apps/client/src/shared/router/router.tsx

```diff
@@ -0,0 +1,17 @@
+import { Suspense } from 'react';
+import { HomePage } from './lazy';
+import { Route, Routes } from 'react-router-dom';
+import GlobalLayout from './global-layout';
+import { routePath } from '@shared/constants/path';
```

Comment:
```suggestion
import { Suspense } from 'react';
import { Route, Routes } from 'react-router-dom';

import GlobalLayout from './global-layout';
import { HomePage } from './lazy';
import { routePath } from '@shared/constants/path';
```
외부라이브러리, 상대경로 import, 절대경로 import를 기준으로 import문을 분리해주면 가독성에 더 좋을 것 같아요 !

Reply:
수정했습니다 😀